### ISSUE TITLE
Two "updating NVDA" dialogs could be shown

### ISSUE THREAD

**Issue by LeonarddeR** [nvaccess/nvda#17931]
### Steps to reproduce:
1. Open a recent NVDA launcher
2. Let it install, ensure that the "Updating NVDA" dialog is shown with the indeterminate progress beeps
3. Quickly open the secure desktop
4. In the background, updating NVDA fails because the secure desktop holds the executable
5. Close the secure desktop and press Retry in the prompt that is shown.

### Actual behavior:
Rapidly pressing alt+tab reveals that there are now two instances open of the updating dialog.

### Expected behavior:
Only one updating dialog should be shown.

#### NVDA installed/portable/running from source:
Installing from launcher

#### NVDA version:
alpha-35993,ec1f161b

#### Windows version:
Windows 11 24H2

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, I've seen this earlier, but I'm not sure when it started.

#### If NVDA add-ons are disabled, is your problem still occurring?
Not tested yet, I'm not sure how to do this from the launcher.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
@LeonarddeR is this not a duplicate of #4579?

**Comment by LeonarddeR** [comment]
No. #4579 deals with the dialog that offers the update. This issue deals with the dialog that is shown in the launcher when updating.
